### DISCUSSION TITLE
Best way to migrate data from one MLFlow tracking server database to another backed by AWS Aurora MYSQL

### DISCUSSION THREAD

**Discussion by ekam-zf** [mlflow/mlflow#10267] [Q&A]
Hi

I have 3 MLFlow tracking servers running over ECS and using combination of AWS Aurora MYSQL and AWS RDS MYSQL as backend databases.

First two MLFLow servers were for experimental purpose and third one is stable and currently tracking live experiments.

I want to consolidate data from old experimental databases (RDS MYSQL & Aurora Serverless MYSQL) to the current stable one (AURORA Serverless MYSQL). So that I can delete the two experimental servers and Databases.

I was thinking on using AWS DMS service for the purpose, however I wanted to know whether this is the best way to get all the data from old experimental databases to the stable one.

I am looking to get all the experiments, runs , metrics etc from experimental databases to be visible in the stable one.

I encountered this import/export tools , however looks like it supports only MLFLow databricks based servers whereas I am using ECS Fargate server + AWS Aurora Mysql combination.

[Import/Export Tool](https://github.com/mlflow/mlflow-export-import)

Thanks

**Comment by andremesarovic** [comment]
The tool only uses the MLflow public API. The backing databases shouldn't matter. The tool supports both Databricks and OSS MLflow.
